packaging/technical-addon/Splunk_TA_otel/windows_x86_64/bin/Splunk_TA_otel.cmd

```diff
@@ -192,11 +185,11 @@ call :extract_bundle
 set "command_line=%splunk_TA_otel_app_directory%%splunk_otel_process_name%"
 call :splunk_TA_otel_log_msg "INFO" "Terminating any orphaned instances of the collector (if present)..."
 start /B "" /I /WAIT powershell -ExecutionPolicy ByPass "& { Get-WmiObject Win32_Process | ? ProcessName -eq '%splunk_otel_process_name%' | ? CommandLine -like '*%command_line%*' | ForEach-Object { taskkill.exe /f /pid $_.ProcessId } }" >> "%splunk_TA_otel_log_file%"
-call :splunk_TA_otel_log_msg "INFO" "Starting otel agent with: %command_line% %SPLUNK_OTEL_FLAGS%"
-start /B "" "%command_line%" %SPLUNK_OTEL_FLAGS% > "%splunk_otel_log_file_value%" 2>&1
 set "splunk_otel_common_ps=%splunk_TA_otel_app_directory%%splunk_otel_common_ps_name%"
 call :splunk_TA_otel_log_msg "INFO" "Monitoring collector process with: %splunk_otel_common_ps% '%splunk_otel_process_name%'"
-start /B "" /I /WAIT powershell -ExecutionPolicy ByPass "& '%splunk_otel_common_ps%' '%splunk_otel_process_name%'" >> "%splunk_TA_otel_log_file%"
+start /B "" /I powershell -ExecutionPolicy ByPass "& '%splunk_otel_common_ps%' '%splunk_otel_process_name%'" >> "%splunk_TA_otel_log_file%"
+call :splunk_TA_otel_log_msg "INFO" "Starting otel agent with: %command_line% %SPLUNK_OTEL_FLAGS%"
+"%command_line%" %SPLUNK_OTEL_FLAGS%
```

Comment:
Why was the command moved here?

Reply:
See above, notice that the `call :splunk_TA_otel_log_msg` is a log statement